app/controllers/webauth_controller.rb

```diff
@@ -18,10 +18,10 @@ def login
   # TODO: can't think of a reasonable way to do strong params here
 
   def login_file
-    redirect_to file_path(params.symbolize_keys)
+    redirect_to file_path(params.to_unsafe_h.symbolize_keys)
   end
 
   def login_iiif
-    redirect_to iiif_path(params.symbolize_keys)
+    redirect_to iiif_path(params.to_unsafe_h.symbolize_keys)
```

Comment:
Exactly; future work should include using strong params instead. In the meantime, we'll at least be aware of the danger (if any; this usage seems fairly innocuous, I think)